### DISCUSSION TITLE
no results with the search box and search results webart

### DISCUSSION THREAD

**Discussion by jurivero** [microsoft-search/pnp-modern-search#3097] [Q&A]
### Check existing issues before logging new ones
If you are not using the latest version, triage your issue with the latest version before logging as we won't provide patches to older versions.

### Issues related to data mapping, web part setup etc, and SharePoint columns to crawled properties to managed properties should be asked in the discussion section.

**For any question regarding Web Parts usage, please use the ["Discussions"](https://github.com/microsoft-search/pnp-modern-search/discussions) tab instead of creating a new issue.**.

__Remove the above section before submitting, and removal of the below template will result in the issue being closed.__

**Version used**
Ex: 4.8

**Describe the bug**
A clear and concise description of what the bug is. Keep questions and bugs related to the web parts. If the issue relates to KQL or question around crawled and managed properties please use [Microsoft TechCommunity](https://techcommunity.microsoft.com/t5/sharepoint-developer/bd-p/SharePointDev). Which means, if you experience the same behavior in classic web parts, then it's not an issue with PnP Modern Search 😉

**To Reproduce**
Detailed steps to reproduce the behavior:
4. Enter setting A, B, C
5. See error

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Additional context**

**Comment by jurivero** [comment]
I have a very simple search 
one search box 
and one search results web part
I have documents with the name communication in 3 different libraries, 3 different sites.
below are my 3 different templates:
1st query Template
{searchTerms} {Path:"https://xxx.sharepoint.com/sites/TestC*" OR Path:"https://xxxx.sharepoint.com/sites/TestC1*" OR Path:"https://xxx.sharepoint.com/sites/TestC2*"}
does not return anything

2nd query  template
{searchTerms} returns anything but documents with the name communications

3rd query  Template 
{searchTerms} IsDocument:1 -fileextension:aspx {Path:"https://xxx.sharepoint.com/sites/TestC*" OR Path:"https://xxxx.sharepoint.com/sites/TestC1*" OR Path:"https://xxx.sharepoint.com/sites/TestC2*"}
does not return anything

None of the query templates return.

Any help will be mostly appreciated.

**Comment by kasperbolarsen** [comment]
The first thing I see it that you wrap the path in curly brackets. Try replacing them with ordinary parentheses

**Comment by jurivero** [comment]
This one works
{searchTerms} Path:"https://xxx.sharepoint.com/sites/TestC*"

and
when typing in the search box I use an * at the end of the word

example:
comm*

**Comment by kasperbolarsen** [comment]
Try:

{searchTerms} (Path:"https://xxx.sharepoint.com/sites/TestC*" OR Path:"https://xxxx.sharepoint.com/sites/TestC1*" OR Path:"https://xxx.sharepoint.com/sites/TestC2*")
